README.md

```diff
@@ -72,3 +72,30 @@ For full details on how to use Streamlink visit our documentation at [streamlink
 Feel free to open a bug or contribute to code!
 
 No need to go through a large CONTRIBUTING.md doc. The only requirement being that we get at least one ACK from a maintainer. Fork / clone us and open a PR!
+
+<a href="graphs/contributors"><img src="https://opencollective.com/streamlink/contributors.svg?width=890" /></a>
```

Comment:
I'd suggest creating a different section with the header "Contributors" and putting this here.

Otherwise, it looks good (from https://github.com/opencollective/streamlink/blob/e6d02455a477da9c78550317c42a38065bf4ec20/README.md )